modules/highgui/include/opencv2/highgui.hpp

```diff
@@ -211,24 +213,27 @@ enum { EVENT_FLAG_LBUTTON   = 1,
        EVENT_FLAG_ALTKEY    = 32
      };
 
-// Qt font
-enum {  QT_FONT_LIGHT           = 25, //QFont::Light,
-        QT_FONT_NORMAL          = 50, //QFont::Normal,
-        QT_FONT_DEMIBOLD        = 63, //QFont::DemiBold,
-        QT_FONT_BOLD            = 75, //QFont::Bold,
-        QT_FONT_BLACK           = 87  //QFont::Black
+//! Qt font
+enum QtFontWeights {
+        QT_FONT_LIGHT           = 25, //!< QFont::Light ( Weight of 25 )
```

Comment:
as referenced by http://docs.opencv.org/modules/highgui/doc/qt_new_functions.html#fontqt i did. but not sure if it needed. maybe QFont::Light is enough